domain/timeline/src/commonMain/kotlin/social/androiddev/domain/timeline/model/StatusUI.kt

```diff
@@ -9,15 +9,21 @@
  */
 package social.androiddev.domain.timeline.model
 
-data class Status(
-    val id: String,
+import social.androiddev.domain.timeline.FeedType
+
+data class StatusUI(
```

Comment:
Should "UI" models really exist in the domain layer? Or is this just a naming convention? 

I've been bitten in the past before when reference domain models in the UI layer itself. (And instead having a ViewModel or mapper transform domain model -> view/ui specific model.

Reply:
naming, fixed to StatusLocal to denote it being the local representation